README.md

```diff
@@ -79,7 +80,7 @@ This variable holds a collection of tags grouped by key representing its target
 
 ```tcl
 module "spectral_lambda_integration" {
-  source                        = "github.com/SpectralOps/spectral-terraform-lambda-integration?ref=v1.0.2"
+  source                        = "github.com/SpectralOps/spectral-terraform-lambda-integration"
```

Comment:
what is the meaning of the ref number here?

Reply:
It is a reference for a specific version of the module, better to reference the latest version in the example by not specifying a version at all